containers/go/README.md

```diff
@@ -14,7 +14,13 @@
 
 ## Using this definition with an existing folder
 
-This definition does not require any special steps to use. Just follow these steps:
+While the definition itself works unmodified, you can select the version of Go the container uses by updating the `VARIANT` arg in the included `devcontainer.json` (and rebuilding if you've already created the container).
+
+```json
+"args": { "VARIANT": "1.13" }
```

Comment:
let's suggest 1.14 :-) (soon we will be with 1.15)

Reply:
Done!